code/cvss/src/Security/CVSS40Lookup.hs

```diff
@@ -0,0 +1,325 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Security.CVSS40Lookup (lookupScore, maxComposed, maxComposedEQ3, maxSeverityeq3eq6, maxSeverity) where
```

Comment:
(just a code organisation thing) - I would like this module to be beneath `Security.CVSS`, e.g. `Security.CVSS.V4_0`.

Reply:
@frasertweedale done